models/DataObject/ClassDefinition/Data/AdvancedManyToManyObjectRelation.php

```diff
@@ -857,14 +857,17 @@ public function enrichLayoutDefinition(/* ?Concrete */ $object, /* array */ $con
                     $this->visibleFieldDefinitions[$field]['title'] = $translator->trans($field, [], 'admin');
                     $this->visibleFieldDefinitions[$field]['fieldtype'] = 'input';
                 }
-            } else {
+            }
+            else {
                 $this->visibleFieldDefinitions[$field]['name'] = $fd->getName();
                 $this->visibleFieldDefinitions[$field]['title'] = $fd->getTitle();
                 $this->visibleFieldDefinitions[$field]['fieldtype'] = $fd->getFieldType();
                 $this->visibleFieldDefinitions[$field]['noteditable'] = true;
 
-                if ($fd instanceof DataObject\ClassDefinition\Data\Select || $fd instanceof DataObject\ClassDefinition\Data\Multiselect) {
-                    if ($fd->getOptionsProviderClass()) {
+                if ($fd instanceof DataObject\ClassDefinition\Data\Select ||
```

Comment:
Preferably format similar to (for readability):
```php
if (
    $fd instanceof DataObject\ClassDefinition\Data\Select
    || $fd instanceof DataObject\ClassDefinition\Data\Multiselect
    || $fd instanceof DataObject\ClassDefinition\Data\BooleanSelect
) {
```

Reply:
As a sidenote: this is from the PSR-12 standard. I'm not sure if Pimcore is using this.
https://www.php-fig.org/psr/psr-12/#51-if-elseif-else

> Expressions in parentheses MAY be split across multiple lines, where each subsequent line is indented at least once. When doing so, the first condition MUST be on the next line. The closing parenthesis and opening brace MUST be placed together on their own line with one space between them. Boolean operators between conditions MUST always be at the beginning or at the end of the line, not a mix of both.

Reply:
@Corepex Thanks for the fixes.